src/lobby/react.js

```diff
@@ -32,6 +38,12 @@ import LobbyCreateRoomForm from './create-room-form';
  *   A React component that provides a UI to create, list, join, leave, play or spectate game instances.
  */
 
+// TODO add option to long-polling like <Lobby polling={{ period: 5000 }} ... />
```

Comment:
Sorry for many todos. I just don't think it's a good idea to create github issues before any discussions.

Reply:
I've copied these over to #354.  You can remove these from this PR and we can discuss there.

Reply:
Thanks!